xamarin/Xamarin.Forms
# [Android/iOS] RefreshView

### Description of Change ###

[RefreshView](https://github.com/xamarin/Xamarin.Forms/issues/5882) represents a container control that provides **pull-to-refresh** functionality for **scrollable** content.

fixes #5882

### Screenshots

Android
![refreshview-droid](https://user-images.githubusercontent.com/6755973/63422982-b01ebe80-c40b-11e9-83a4-afb2671f4c93.gif)

iOS
![refreshview-ios2](https://user-images.githubusercontent.com/6755973/63512212-846f0780-c4e3-11e9-8b95-6db3a1f90ef2.gif)

### API Changes ###

```
public class RefreshView : ContentView
{
	public static readonly BindableProperty CommandProperty;
	public ICommand Command { get; set; }

	public static readonly BindableProperty CommandParameterProperty;
	public object CommandParameter { get; set; }

	public static readonly BindableProperty IsEnabledProperty;
	public bool IsEnabled { get; set; }

	public bool IsRefreshing { get; set; }

	public Color BackgroundColor {get; set;}
	public Color RefreshColor {get; set;}
}
```
 
### Platforms Affected ### 


- Core/XAML (all platforms)
- iOS
- Android

### Behavioral/Visual Changes ###


None

### Before/After Screenshots ### 


Not applicable

### Testing Procedure ###
In Core Gallery access to the RefreshView samples. Try to pull to refresh the content. A load indicator should appear and refresh the content.

### PR Checklist ###
